blocks/table/table.js

```diff
@@ -9,20 +9,22 @@ function buildCell(rowIndex) {
   if (!rowIndex) cell.setAttribute('scope', 'col');
   return cell;
 }
-
 export default async function decorate(block) {
   const table = document.createElement('table');
   const thead = document.createElement('thead');
   const tbody = document.createElement('tbody');
   table.append(thead, tbody);
   [...block.children].forEach((child, i) => {
     const row = document.createElement('tr');
-    if (i) tbody.append(row);
-    else thead.append(row);
+    const td = document.createElement('td');
+    const colspan = document.createElement('colspan');
```

Comment:
colspan is not an element. You have to set [attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/td#attributes) `td.setAttribute('colspan', <some-value>)`